docs: Navbar doc fixes (#1511)
bootstrap-vue-next/bootstrap-vue-next: apps/docs/src/docs/components/navbar.md

## Color schemes

`BNavbar` supports the standard Bootstrap v5 available background color variants. Set the `variant` prop to one of the
following values to change the background color: primary, secondary, success, info, warning, danger, dark, light, or any of the \*-subtle variants.

If the `variant` property is set, it may be necessary to control the text color so that it contrasts with the variant
irrespective of the theme. This can be done in one of two ways.

Use the `v-b-color-mode` directive:

> ```vue-html

Reviewer: Code blocks need to be in highlight cards

Author: I believe this is now addressed.

## Color schemes

`BNavbar` supports the standard Bootstrap v5 available background color variants. Set the `variant` prop to one of the
following values to change the background color: primary, secondary, success, info, warning, danger, dark, light, or any of the \*-subtle variants.

If the `variant` prop is set, it may be necessary to control the text color so that it contrasts with the variant
irrespective of the theme. This can be done by use the `v-b-color-mode` directive or `useColorMode` composable

> An example using `v-b-colorMode`. Form more information see the [BColorMode directive](/docs/directives/BColorMode).

Reviewer: ```suggestion
An example using `v-b-colorMode`. For more information see the [BColorMode directive](/docs/directives/BColorMode).
```

<HighlightCard>
  <BNavbar variant="primary" v-b-color-mode="'dark'">
    <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
  </BNavbar>

<template #html>

```vue-html
<BNavbar variant="primary" v-b-color-mode="'dark'">
  <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
</BNavbar>
```

  </template>

> </HighlightCard>

Reviewer: ```suggestion
</HighlightCard>

```

## Color schemes

`BNavbar` supports the standard Bootstrap v5 available background color variants. Set the `variant` prop to one of the
following values to change the background color: primary, secondary, success, info, warning, danger, dark, light, or any of the \*-subtle variants.

> If the `variant` property is set, it may be necessary to control the text color so that it contrasts with the variant

Reviewer: ```suggestion
If the `variant` prop is set, it may be necessary to control the text color so that it contrasts with the variant
```

> irrespective of the theme. This can be done by use the `v-b-color-mode` directive or the `data-bs-theme` attribute.

Reviewer: ```suggestion
irrespective of the theme. This can be done by use the `v-b-color-mode` directive or `useColorMode` composable
```

An example using `v-b-colorMode`. Form more information see the [BColorMode directive](/docs/directives/BColorMode).

<HighlightCard>
  <BNavbar variant="primary" v-b-color-mode="'dark'">
    <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
  </BNavbar>

<template #html>

```vue-html
<BNavbar variant="primary" v-b-color-mode="'dark'">
  <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
</BNavbar>
```

  </template>
</HighlightCard>

> 
> An example using `data-bs-theme`. For more information see the
> [bootstrap documentation](https://getbootstrap.com/docs/5.3/components/navbar/#color-schemes).
> 
> <HighlightCard>
>   <BNavbar variant="primary" data-bs-theme="dark">
>     <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
>   </BNavbar>
>   <template #html>
> 
> ```vue-html
> <BNavbar variant="primary" data-bs-theme="dark">
>   <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
> </BNavbar>
> ```
> 
>   </template>
> </HighlightCard>
> 

Reviewer: ```suggestion
```

following values to change the background color: primary, secondary, success, info, warning, danger, dark, light, or any of the \*-subtle variants.

If the `variant` property is set, it may be necessary to control the text color so that it contrasts with the variant

> irrespective of the theme. This can be done by use the `v-b-color-mode` directive or the `data-bs-theme` attribute.

Reviewer: This doesn't reference `useColorMode`

Author: I believe this is addressed by `BNavbar` will conform to the current color theme if the `variant` prop is not set. If you'd like me to explicitly include how you set the current theme and put a link to useColorMode I will.

<HighlightCard>
  <BNavbar variant="primary" v-b-color-mode="'dark'">
    <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
  </BNavbar>
  <template #html>

```vue-html
<!-- Using the v-b-color-mode directive -->

> <BNavbar variant="faded">

Reviewer: The example for the template needs to match the codeblock from above

Reviewer: The comment is unnecessary. But perhaps you can label it actually above each `<HighlightCard>` above

```html
With `v-b-colorMode` more info [here](#pathToVBColorMode)

<HighlightCard>
...
```

etc etc for each example

Author: I've moved these to text above the cards with links.

  <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
</BNavbar>
```

> </template>

Reviewer: Indent

Author: This should now be fixed.

</HighlightCard>

<HighlightCard>
  <BNavbar variant="primary" data-bs-theme="dark">
    <BNavbarBrand tag="h1" class="mb-0">BootstrapVue</BNavbarBrand>
  </BNavbar>
  <template #html>

```vue-html
<!-- Using the data-bs-theme attribute -->

> <BNavbar data-bs-theme="dark">

Reviewer: The example for the template needs to match the codeblock from above

Author: This is now fixed
